src/Ethernet.h

```diff
@@ -253,11 +258,18 @@ class EthernetClient : public Client {
 class EthernetServer : public Server {
 private:
 	uint16_t _port;
+	// 20190421 https://kmpelectronics.eu/ Plamen Kovandjiev - We added support for ESP32.
+	void init();
 public:
 	EthernetServer(uint16_t port) : _port(port) { }
 	EthernetClient available();
 	EthernetClient accept();
+	// 20190421 https://kmpelectronics.eu/ Plamen Kovandjiev - We added support for ESP32.
+#ifdef ESP32
+	virtual void begin(uint16_t port = 0);
```

Comment:
Related: https://github.com/arduino-libraries/Ethernet/issues/88
It seems like it would be better for the ESP32 core developers to follow the standard Arduino core API, rather than forcing every library to be patched to deal with the breakage they caused.

Reply:
Agreed, the ESP folks appear to have strayed from the Arduino API.  Not good.